### PR TITLE
Update UI Component for the save, edit and re-use report story

### PR DESCRIPTION
**Pull request by jess0407**
- include a new type `saveScheduled` in rb-fieldset
- use a new set of svg icons
- fix bug in rb-check-group
- include angular material

https://rockabox.tpondemand.com/entity/12046
https://rockabox.tpondemand.com/entity/12194

### CODE REVIEW

**Comment by nwhite89** [comment]
Looks good

**Comment by jess0407** [comment]
@nwhite89 cool, label added

File: src/rb-fieldset/rb-fieldset.css [hunk 1]
```diff
@@ -76,7 +76,7 @@
     background-image: url("./images/additional-info.svg");
     background-position: var(--Fieldset-title-background-position);
     background-repeat: no-repeat;
-    padding-left: 32px;
+    padding-left: 40px;
```
Comment (nwhite89) [inline src/rb-fieldset/rb-fieldset.css]: is it worth putting this in a variable? `--Fieldset-title-padding`

**Comment by nwhite89** [comment]
over to @rockabox/qa

File: src/rb-fieldset/rb-fieldset.css (hunk 1 above)
Comment (jaylett) [inline src/rb-fieldset/rb-fieldset.css]: `--Fieldset-title-padding-left`?

**Comment by jim-dj** [comment]
Can be merged for QA

**Pull request merged**